### PR TITLE
[O2B-1080] home overviews serparation

### PR DESCRIPTION
**Pull request by xsalonx**
#### I have a JIRA ticket
- [x] branch and/or PR name(s) include(s) JIRA ID
- [x] issue has "Fix version" assigned
- [x] issue "Status" is set to "In review"
- [x] PR labels are selected

Notable changes for users:
- NA

Notable changes for developers:
- use separate model for Home page
- reduce number of request of RunsOverviewModel

Changes made to the database:
- NA

### CODE REVIEW

File: lib/public/view.js [hunk 1]
```diff
@@ -35,6 +34,7 @@ import { TagDetailsPage } from './views/Tags/Details/TagDetailsPage.js';
 import { EosReportCreationPage } from './views/EosReport/create/EosReportCreationPage.js';
 import { StatisticsPage } from './views/Statistics/StatisticsPage.js';
 import { LhcPeriodsOverviewPage } from './views/lhcPeriods/Overview/LhcPeriodsOverviewPage.js';
+import { HomeOverviewPage } from './views/Home/Overview/index.js';
```
Comment (martinboulais) [inline lib/public/view.js:37]: I think you should simply name this HomePage

File: lib/public/views/Home/Overview/HomeModel.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+ */
+
+import { Observable } from '/js/src/index.js';
+import { RunsOverviewModel } from '../../Runs/Overview/RunsOverviewModel.js';
+import { LogsOverviewModel } from '../../Logs/Overview/LogsOverviewModel.js';
+
+/**
+ * Model representing handlers for home page
+ */
+export class HomeModel extends Observable {
```
Comment (martinboulais) [inline lib/public/views/Home/Overview/HomeModel.js:21]: I would have named it HomePageModel

File: lib/public/views/Home/Overview/HomeModel.js [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+ * granted to it by virtue of its status as an Intergovernmental Organization
+ * or submit itself to any jurisdiction.
+ */
+
+import { Observable } from '/js/src/index.js';
+import { RunsOverviewModel } from '../../Runs/Overview/RunsOverviewModel.js';
+import { LogsOverviewModel } from '../../Logs/Overview/LogsOverviewModel.js';
+
+/**
+ * Model representing handlers for home page
```
Comment (martinboulais) [inline lib/public/views/Home/Overview/HomeModel.js:19]: handlers?

**Review by martinboulais — CHANGES_REQUESTED** [review]

File: lib/public/views/Home/Overview/HomeModel.js (hunk 3 above)
Comment (xsalonx) [reply lib/public/views/Home/Overview/HomeModel.js]: changed

File: lib/public/views/Home/Overview/HomeModel.js (hunk 2 above)
Comment (xsalonx) [reply lib/public/views/Home/Overview/HomeModel.js:21]: renamed

File: lib/public/view.js (hunk 1 above)
Comment (xsalonx) [reply lib/public/view.js]: renamed

**Review by martinboulais — APPROVED** [review]

**Review by martinboulais — APPROVED** [review]

**Pull request merged**